variables.tf

```diff
@@ -20,6 +20,40 @@ variable "assign_generated_ipv6_cidr_block" {
   default     = false
 }
 
+variable "enable_ipv6" {
+  description = "Assigns IPv6 subnets and routes"
+  default     = false
+}
+
+variable "private_subnet_ipv6_prefixes" {
```

Comment:
Strictly speaking.. those are not prefixes, but more the.. indices of the prefixes? Suggestions?

Reply:
Agreed, they aren't. They are the last octet of the given ipv6 prefix. IPv6 docs tends to be prefix based, so I chose that. Admittedly, it's not a great name, but I couldn't for the life of me figure out what a better way to describe the fact they are an enumerated list of the last octet in an IPv6 prefix such as `2001:db8::ff00:42:83XX`. I explicitly didn't go with index or slice because I didn't want it to be confused as an index of another list.

So maybe `..._subnet_ipv6_octet`?

Reply:
What about giving an example in the description? While it already suggests 0-256, an example like `[0, 1, 2]`might make it clearer. The description (and the topic) feels pretty complex :-)